lblod/app-lokaal-mandatenbeheer
# Move from mu-authorization to sparql-parser

## Description

Using the old mu-authorization config is causing some issues with 500 errors, the new semtech/sparql-parser seems to work better in this regard.

## How to test

- See if the apps still works, all resources will be affected by this change.
- cURL to burgemeester benoeming, (with docker-compose.dev.yml turned off) as this was broken with the old mu-auth
